### ISSUE TITLE
glimmer scaffold:customcontrol[model_form] not working

### ISSUE THREAD

**Issue by MtnBiker** [AndyObtiva/glimmer-dsl-libui#72]
I ran `➜ glimmer "scaffold[photo_processing]"`

and later tried and the rest is just to show the app is working.

`**glimmer scaffold:customcontrol[model_form]**
zsh: no matches found: scaffold:customcontrol[model_form]

photo_processing on  main [!] via 💎 ruby-3.4.1 
➜ glimmer run
Launching Glimmer Application: ./bin/photo_processing
Launching Glimmer Application: /Users/gscar/Documents/Ruby/photo_processing/app/photo_processing/launch.rb
2024-12-26 17:33:09.636 ruby[70704:11837380] +[IMKClient subclass]: chose IMKClient_Modern
2024-12-26 17:33:09.636 ruby[70704:11837380] +[IMKInputSession subclass]: chose IMKInputSession_Modern`

Am I missing something? New to Glimmer and not very skilled in Ruby.

**Comment by AndyObtiva** [comment]
On my Mac ARM64 (M2 Chip) Sequoia 15.0.1, I ran this command:

```
glimmer "scaffold[photo_processing]"
```

It generated the app and ran it successfully.

![Screenshot 2024-12-26 at 8 55 36 PM](https://github.com/user-attachments/assets/3d56b8b2-5586-4a74-893d-3f8c5f2a3a31)

I quit the app, and then I entered the directory of the app:

```
cd photo_processing
```

Next, I ran this command to launch the app again:

```
glimmer run
```

It started the app again, successfully.

![Screenshot 2024-12-26 at 8 55 36 PM](https://github.com/user-attachments/assets/3d56b8b2-5586-4a74-893d-3f8c5f2a3a31)

So, I am unable to reproduce the issue you encountered.

However, we might be on different platforms and operating systems.

What OS are you running? Mac, Windows, or Linux? What version is it exactly? What CPU (processor) do you have (Intel or ARM)? 

 > and later tried and the rest is just to show the app is working.

That reads like broken English. Could you please say again what you were saying?

> zsh: no matches found: scaffold:customcontrol[model_form]

Regarding that error? It happens under zsh (but not bash) if someone runs a Glimmer command without double-quoting the subcommand that follows `glimmer` (we need to double-quote `scaffold:customcontrol[model_form]` as `"scaffold:customcontrol[model_form]"`).

Here is the correct way of running this command, which I tried inside `photo_processing` and it worked:
```
% glimmer "scaffold:customcontrol[model_form]"
Created photo_processing/app/photo_processing/view/model_form.rb
```

**Comment by MtnBiker** [comment]
Thank you Andy for the quick response. I had just figured that out for myself by setting up a new app and noticing the quotes. But the section on Scaffolding is missing the quotes for these commands. I now see that you clarified that in your response. macOS M2

My "broken English" part was just to show that Glimmer was working. I copied from iTerm. Sorry for the confusion.

Looking forward to working with Glimmer. Looks like what I wanted. Replacing Pashua while updating a personal script. I think I'll make an app now.
